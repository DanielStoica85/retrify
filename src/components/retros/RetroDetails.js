import React from 'react';
import PropTypes from 'prop-types';

const RetroDetails = props => {
    const id = props.match.params.id;
    return (
        <div className="container section project-details">
            <div className="card">
                <div className="card-content">
                    <span className="card-title">Retro Title #{id} </span>
                    <p>
                        Lorem ipsum dolor sit amet consectetur adipisicing elit.
                        Perspiciatis velit ab optio eius corporis iste facilis,
                        sit eum distinctio inventore repudiandae obcaecati
                        eveniet sunt, quisquam iure quaerat amet reiciendis
                        error?
                    </p>
                </div>
                <div className="card-action grey lighten-4 grey-text">
                    <div>Posted by Daniel</div>
                    <div>May 13th, 2am</div>
                </div>
            </div>
        </div>
    );
};

RetroDetails.propTypes = {
    match: PropTypes.object
};

export default RetroDetails;
