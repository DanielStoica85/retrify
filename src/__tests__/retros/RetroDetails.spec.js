import React from 'react';
import renderer from 'react-test-renderer';
import RetroDetails from '../../components/retros/RetroDetails';

describe('RetroDetails component', () => {
    const props = {
        match: {
            params: {
                id: 123
            }
        }
    };
    it('renders correctly', () => {
        const component = renderer.create(<RetroDetails {...props} />).toJSON();
        expect(component).toMatchSnapshot();
    });
});
