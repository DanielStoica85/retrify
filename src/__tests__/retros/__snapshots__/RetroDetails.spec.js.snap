// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RetroDetails component renders correctly 1`] = `
<div
  className="retro-details container"
>
  <div
    className="card"
  >
    <div
      className="card-body"
    >
      <div
        className="card-title"
      >
        Retro Title #
        123
      </div>
      <p
        className="card-text"
      >
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Perspiciatis velit ab optio eius corporis iste facilis, sit eum distinctio inventore repudiandae obcaecati eveniet sunt, quisquam iure quaerat amet reiciendis error?
      </p>
    </div>
    <div
      className="card-footer"
      style={
        Object {
          "backgroundColor": "lightgray",
          "borderColor": "lightgray",
        }
      }
    >
      <div>
        Posted by Daniel
      </div>
      <div>
        May 13th, 2am
      </div>
    </div>
  </div>
</div>
`;
